### ISSUE TITLE
Are wildcard media types supported in content?

### ISSUE THREAD

**Issue by hkosova** [OAI/OpenAPI-Specification#1290]
The 3.0 Specification contains examples where the request and response `content` includes wildcard media types, e.g. here:
https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#request-body-examples
https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#path-item-object-example
``` yaml
description: user to add to the system
content: 
  'application/json':
    schema:
      $ref: '#/components/schemas/User'
  ...
  '*/*':
    examples:
      user: 
        summary: User example in other format
        externalValue: 'http://foo.bar/examples/user-example.whatever'
```

Are these examples valid and wildcard types actually supported?

The description of `content` says just this (emphasis mine):
>content
>Map[string, Media Type Object]
>REQUIRED. The content of the request body. **The key is the media type** and the value describes it.

For comparison, the [Encoding Object](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#encoding-object) explicitly states that wildcard types are allowed:
>contentType
>... The value can be a specific media type (e.g. `application/json`), a wildcard media type (e.g. `image/*`), or a comma-separated list of the two types.

If wildcard types are supported in `content`, how does this work? Do more specific types have preference over wildcard ones, e.g. `image/png` > `image/*` > `*/*`?

**Comment by webron** [comment]
Perhaps we shouldn't have been explicit in the Encoding Object, and perhaps we should have been more explicit elsewhere. Wildcard media type is a standard valid value. 

As for the processing order, what you wrote seems right. I don't know if these rules are provided by any other RFC or if they should be added to the spec (/cc @darrelmiller).

**Comment by darrelmiller** [comment]
For the encoding object, when the `contentType` property identifies more than one possible media type, it is effectively saying that any of the matching media types may be present in the multi-part document.  It is expected that the content-type header field in the actual request or response will describe the exact media type being sent.  I do not believe there is a need to describe any kind of precedence.

**Comment by webron** [comment]
Fixed by #1295.
